docs/variants/clevo_nv41/fwupd_usage.md

```diff
@@ -159,21 +170,23 @@ An update requires the system to shutdown to complete. Shutdown now? [y|N]:
 
 3. Power on the laptop again
 4. Log in and run the following command to verify results:
+
 ```bash
 $ sudo fwupdmgr get-results
 
 Choose a device:
-0.	Cancel
-1.	4bde70ba4e39b28f9eab1628f9dd6e6244c03027 (11th Gen Intel Core™ i7-1165G7 @ 2.80GHz)
-2.	5792b48846ce271fab11c4a545f7a3df0d36e00a (Display controller)
-3.	073c01931cb0e9889bbfb2ea4a4c2fc558805fc6 (Display controller)
-4.	dbee8bd3b1ae0316ad143336155651eedb495a0e (NV4XMB,ME,MZ)
-5.	71b677ca0f1bc2c5b804fa1d59e52064ce589293 (SSD 980 PRO 1TB)
-6.	c6a80ac3a22083423992a3cb15018989f37834d6 (TPM)
-7.	eefcbd318bd31fc1eba6358e628b3f9dceb87206 (USB4 host controller)
+0. Cancel
+1. 4bde70ba4e39b28f9eab1628f9dd6e6244c03027 (11th Gen Intel Core™ i7-1165G7 @ 2.80GHz)
+2. 5792b48846ce271fab11c4a545f7a3df0d36e00a (Display controller)
+3. 073c01931cb0e9889bbfb2ea4a4c2fc558805fc6 (Display controller)
+4. dbee8bd3b1ae0316ad143336155651eedb495a0e (NV4XMB,ME,MZ)
+5. 71b677ca0f1bc2c5b804fa1d59e52064ce589293 (SSD 980 PRO 1TB)
+6. c6a80ac3a22083423992a3cb15018989f37834d6 (TPM)
+7. eefcbd318bd31fc1eba6358e628b3f9dceb87206 (USB4 host controller)
```

Comment:
same here, this is output pasted from console, maybe visually-wise it doesn't make difference

Reply:
disabled for code blocks